charts/dependentapi-simple-operator/templates/rbac.yaml

```diff
@@ -39,6 +39,10 @@ rules:
     resources: [components]
     verbs: [list, watch, get, patch]
 
+  - apiGroups: [oda.tmforum.org]
+    resources: [apis]
+    verbs: [list, watch, get, patch]
```

Comment:
Is "patch" really needed? I think APIs should be accessed read-only.

Reply:
revoke Patch access for API CR